resnet50_paddle_cloud_dist/fluid/fluid_benchmark.py

```diff
@@ -360,6 +374,13 @@ def main():
                 "Must configure correct environments to run dist train.")
         train_args.extend([train_prog, startup_prog])
         if args.gpus > 1 and os.getenv("TRAINING_ROLE") == "TRAINER":
+            max_count = 100
+            time.sleep(60 * 2)
```

Comment:
no need for this?

Reply:
aligned with QA, ```sleep``` is needed for ```PaddleCloud```

Reply:
but you have retry below? Maybe it doesn't need 120 sec?

Reply:
we should wait at least 120 secs ...

Reply:
if you retry every 60 secs. And retry 10 times, it would be 600secs?...